gravitee-apim-e2e/api-test/src/apis/imports/mapi-v2/import-api-definition/api-v4-import-with-members.spec.ts

```diff
@@ -106,14 +85,6 @@ describe('API - V4 - Import - Gravitee Definition - With members', () => {
     });
 
     test('should get API members, with a primary owner, and an additional member with custom role', async () => {
-      // API has been imported with a primary owner, so default API Publisher used to import should not be able to access it
```

Comment:
This rule is not true anymore. This has been discussed and validated by the product team